File: sdkwork-app-sdk-csharp/Models/GenerationTypeDistributionVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class GenerationTypeDistributionVO
    {
        public double? Text { get; set; }
        public double? Image { get; set; }
        public double? Audio { get; set; }
        public double? Video { get; set; }
        public double? Code { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/generation-type-distribution-vo.ts
/** 生成类型分布 */
export interface GenerationTypeDistributionVO {
  /** 文本生成比例 */
  text?: number;
  /** 图像生成比例 */
  image?: number;
  /** 音频生成比例 */
  audio?: number;
  /** 视频生成比例 */
  video?: number;
  /** 代码生成比例 */
  code?: number;
}
